scripts/perf-run-scenario

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+set -e
+
+SCRIPTNAME=$(basename $0)
+
+if [ $# != 4 ]; then
+    cat << EOF
+Usage: ${SCRIPTNAME} <imagetag> <version> <version> <artifacts>
```

Comment:
if we go with a consistent image tag name in the build script, we should just take the scenario name here instead of an image tag

Reply:
Same reasoning for this as stated in https://github.com/DataDog/dd-trace-py/pull/2770#discussion_r687090620